ext/http/00_serve.ts

```diff
@@ -288,28 +284,28 @@ class InnerRequest {
 
     // * is valid for OPTIONS
     if (path === "*") {
-      return this.#urlValue = "*";
+      return (this.#urlValue = "*");
```

Comment:
Surprised that dprint is not removing these changes to parentheses

Reply:
I accidentally ran prettier - it added these parentheses